### ISSUE TITLE
Compile Error with Angular 10

### ISSUE THREAD

**Issue by FloBam** [mwootendev/ngx-translate-plugins#15]
When running `ng test` after updating to Angular 10:
```
ERROR in node_modules/ngx-translate-testing/lib/testing.module.d.ts:13:56 - error TS2314: Generic type 'ModuleWithProviders<T>' requires 1 type argument(s).

13 export declare class TranslateTestingModule implements ModuleWithProviders {
```

**Comment by paul-kr** [comment]
I submitted PR #16 as a potential fix. When I patch it locally this same way, it works without error.

**Comment by Maximaximum** [comment]
@mwootendev could you please take a look at this?

**Comment by mwootendev** [comment]
@FloBam thank you for opening the ticket. @paul-kr thanks for the fix. @Maximaximum thanks for tagging me, since that is what notified me. I've released version 5.0.0 of ngx-translates-testing that supports Angular 10 and @ngx-translate/core 13.0.0. Please let me know if you have additional issues.
